pkg/utils/database/postgres_test.go

```diff
@@ -24,7 +24,7 @@ import (
 )
 
 func testPostgres() *Postgres {
-	return &Postgres{"local", test.GetPostgresHost(), test.GetPostgresPort(), "testdb", "testuser", "testpassword", []string{}, false, false, false, []string{}}
+	return &Postgres{"local", test.GetPostgresHost(), test.GetPostgresPort(), "testdb", "testuser", "testpassword", false, []string{}, false, false, false, []string{}}
```

Comment:
think this has too many positional arguments, would go with an explicit key:value struct creation (if possible) for visibility.